tracer/test/Datadog.Trace.Tools.dd_dotnet.ArtifactTests/Utils.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="Utils.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Datadog.Trace.TestHelpers;
+
+namespace Datadog.Trace.Tools.dd_dotnet.ArtifactTests;
+
+public static class Utils
+{
+    internal static string GetApiWrapperPath()
+    {
+#if NETFRAMEWORK
+        return string.Empty;
+#else
+        string archFolder;
+
+        if (FrameworkDescription.Instance.ProcessArchitecture == ProcessArchitecture.Arm64)
+        {
+            archFolder = "linux-arm64";
+        }
+        else
+        {
+            archFolder = IsAlpine() ? "linux-musl-x64" : "linux-x64";
+        }
+
+        return Path.Combine(EnvironmentHelper.GetMonitoringHomePath(), archFolder, "Datadog.Linux.ApiWrapper.x64.so");
+#endif
+    }
+
+    internal static string GetDdDotnetPath()
+    {
+        var rid = (EnvironmentTools.GetOS(), EnvironmentTools.GetPlatform(), EnvironmentHelper.IsAlpine()) switch
+        {
+            ("win", _, _) => "win-x64",
+            ("linux", "Arm64", _) => "linux-arm64",
+            ("linux", "X64", false) => "linux-x64",
+            ("linux", "X64", true) => "linux-musl-x64",
+            _ => throw new PlatformNotSupportedException()
+        };
+
+        return Path.Combine(EnvironmentHelper.GetMonitoringHomePath(), rid, RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "dd-dotnet.exe" : "dd-dotnet");
+    }
+
+    internal static bool IsAlpine()
```

Comment:
In CI, we also just set a variable that you can check 😃

Reply:
Honestly I prefer this way. At least it doesn't break in mysterious way when you run it locally and forget to set the environment variable.